baseline-null-away/build.gradle

```diff
@@ -0,0 +1,7 @@
+apply plugin: 'java-library'
+apply plugin: 'com.palantir.external-publish-jar'
+
+dependencies {
+    runtimeOnly 'com.uber.nullaway:nullaway'
+    runtimeOnly 'org.checkerframework:dataflow-nullaway'
```

Comment:
just to confirm as my gradle-fu is not as knowledgeable here and I haven't dug into the classfiles produced -- this only puts these on the `baseline-null-away` gradle plugin runtime classpath when the plugin is applied and does not trigger a transitive runtime dependency to any modules that apply the plugin itself (i.e. we won't produce any class files that assume `dataflow-nullaway` will be on a service's runtime classpath), is that correct?

Reply:
That's correct, although nothing depends directly on this module. We produce this library so that we can control the version of nullaway via the gradle-baseline versions.props and standard excavation, and add the `baseline-null-away` dependency to the `errorprone` checks dependency configuration using the current baseline version.

Reply:
Initially I attempted to avoid a new module by checking the package implementationVersion of a nullaway class, however that resulted in a larger plugin classpath (not a huge issue) and unfortunately the nullaway jars don't include an `Implementaiton-Version` in their jar manifest, so I couldn't query the version directly. Ultimately I think the solution in this PR is cleaner and more maintainable.